src/main/java/hudson/plugins/copyartifact/TriggeredBuildSelector.java

```diff
@@ -93,16 +95,18 @@ public boolean isForGlobalSetting() {
     };
     private Boolean fallbackToLastSuccessful;
     private final UpstreamFilterStrategy upstreamFilterStrategy;
+    private Boolean allowUpstreamDependencies;
```

Comment:
You can use `boolean` here as it's set to `false` if not configured.
You have to use `Boolean` if:
* You want to know it's not configured and want to run an upgrading process.
* You want to set it to `true` if not configured.